examples/player/simid_player.js

```diff
@@ -131,6 +131,30 @@ class SimidPlayer {
     });
   }
 
+  /**
+   * Closes the ad and SIMID creative.
+   */
+  closeAd() {
+    this.onRequestStop(CreativeMessage.REQUEST_STOP);
+    this.sendLog("Ad closed by player");
```

Comment:
Am I missing something? I don't see a sendLog or onRequestStop method anywhere (or onRequestSkip or onCreativeFaltalError)

Reply:
you don't? They are on line 477, 484, 528

Reply:
Ah yep I was missing something haha, I didn't realize they already existed so I didn't open up the non-changed lines

Reply:
I don't think those are quite the methods you want to call tho.  If you look at those, they each take in an incomingMessage, which they then send to simidProtocol.resolve/reject.  Those are currently called when a message is received, and the message object is the argument.  You are passing in just the message name.  If you run this line, what message eventually gets sent to the creative? Is that the message you want to be sent?

Reply:
fixed to call proper functions